source/standalone/demos/bipeds.py

```diff
@@ -49,9 +49,7 @@ def main():
     """Main function."""
 
     # Load kit helper
-    sim = SimulationContext(
-        sim_utils.SimulationCfg(device="cpu", use_gpu_pipeline=False, dt=0.01, physx=sim_utils.PhysxCfg(use_gpu=False))
-    )
+    sim = SimulationContext(sim_utils.SimulationCfg(device="cpu", dt=0.01))
```

Comment:
Curious why is this demo specifically on CPU? If we aren't already we might highlight this somewhere in docs

Reply:
We do this because when you have few robots in the scene, CPU is usually faster, improving the user experience.